src/postgraphile/http/createPostGraphileHttpRequestHandler.ts

```diff
@@ -519,10 +526,10 @@ export default function createPostGraphileHttpRequestHandler(options) {
     // The `result` will be used at the very end in our `finally` block.
     // Statements inside the `try` will assign to `result` when they get
     // a result. We also keep track of `params`.
-    let paramsList
-    let results
+    let paramsList: any
+    let results: any
```

Comment:
results will be an optional array of GraphQL results (something like `{data?: any, errors?: Array<GraphQLError>}` ?)

Reply:
aa99779

the way the control structures flow here make the compiler (validly, IMHO) think that results can be used before assigned. for this reason i eager assigned results to `[]`.

we should scrub the whole handler generation process at some point and port it into smaller blocks/files/fns/etc